src/Couchbase.Lite.Shared/API/Sync/Replicator.cs

```diff
@@ -364,6 +418,11 @@ private void OnDocError(C4Error error, bool pushing, [NotNull]string docID, bool
                 Log.To.Sync.I(Tag,
                     $"{this}: {transientStr}error {dirStr} '{logDocID}' : {error.code} ({Native.c4error_getMessage(error)})");
             }
+            //TODO - need to handle ended notification
+            if (error.domain == C4ErrorDomain.DocumentEnded && error.code == 0) {
```

Comment:
Just compare the domain to zero.  '0' does not represent 'document ended' so it cannot be named that